### ISSUE TITLE
Additional guidance needed regarding stylesheet.

### ISSUE THREAD

**Issue by Graham-72** [backdrop-contrib/colorbox#7]
The project includes styleheet options in its styles directory.

I found I had to copy the files for one of these options, including images, to my site's theme directory and then add a reference in my theme's info file.

Perhaps this could usefully be explained in the README.md file?

**Comment by jenlampton** [comment]
Yes, we could use some better documentation on how to use existing themes, modify them, or create new ones. 

In your case @Graham-72 were you not able to override the CSS in your theme directly -- without copying the whole style sheet (and images) the colorbox style provided?

**Comment by Graham-72** [comment]
I cannot reproduce the problem I had back in July 2016, nor can I remember the detail. The modules stylesheets and associated image files all seem to work satisfactorily from where they are installed within the module, and the admin configuration facility for selecting which Colorbox style to use is working fine.

**Comment by jenlampton** [comment]
Okay, thanks for testing. Let's leave this issue open and see if others have problems in the future. More documentation is never a bad thing ;)
